spyder/widgets/mixins.py

```diff
@@ -142,7 +144,8 @@ def _get_inspect_shortcut(self):
 
     def _format_text(self, title=None, signature=None, text=None,
                      inspect_word=None, title_color=None, max_lines=None,
-                     display_link=False):
+                     width=_DEFAULT_WIDTH, display_link=False,

```

Comment:
Maybe  `max_width` is a better name as it is more clear what it implies (same as `max_lines`), it could be less than that but never larger. What do you think @dalthviz ?